kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -11,7 +11,7 @@
         {{ /* selectFoldersOrExercises$() */ }}
       </h5>
 
-      <div v-if="!isTopicIdSet && bookmarks.length">
+      <div v-if="!isTopicIdSet && bookmarks.length && !bookMarksFlag">
```

Comment:
Nonblocking and nitpicking a bit -- but "bookmark" is all one word so I think this would read better if it was `bookmarks` throughout.